homeworks/lesson2/app/src/components/page-404/page-404.jsx

```diff
@@ -0,0 +1,13 @@
+import React, {Component} from 'react';
+
+class Page404 extends Component {
+    render() {
```

Comment:
```
const {
   pathname,
} = this.props.location;
```
Please follow to the clean code princeples.
We write code no only for self but and for other developers.
If you using some variable please declare it before using.

Reply:
Thanks, fixed